radix-engine-lib/src/engine/types/third_party.rs

```diff
@@ -6,8 +6,5 @@ pub use sbor::*;
 pub use sbor::*;
 pub use strum::*;
 
-pub use crate::component::{ComponentAddress, PackageAddress};
-pub use crate::core::SystemAddress;
 pub use crate::crypto::Hash;
-pub use crate::resource::NonFungibleId;
-pub use crate::resource::ResourceAddress;
+pub use crate::model::*;
```

Comment:
Maybe it's time to get rid of this file, as we have a dedicated engine interface crate.